lib/osf-components/addon/components/cedar-metadata-editor/styles.scss

```diff
@@ -0,0 +1,45 @@
+// stylelint-disable max-nesting-depth, selector-max-compound-selectors
+
+.cedar-add-container {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+
+    .cedar-container {
+        width: 100%;
+        display: flex;
+        flex-direction: row;
+        justify-content: flex-start;
+        align-items: flex-start;
+        margin-bottom: 30px;
+
+        .cedar-editor {
+            width: 100%;
+
+            h1 {
+                font-size: 24px;
+                font-weight: bold;
+                margin-top: 5px;
+            }
+
+            h2 {
+                display: none;
+            }
+
+        }
+    }
+
+    .action-container {
+        width: calc(100% - 85px);
+        display: flex;
+        flex-direction: row;
+        justify-content: flex-end;
+        align-items: center;
+    }
+}
+
+:global(.template-card) {
```

Comment:
A word of caution when using `:global` without it being nested within a local-class, is that this will apply to _all_ instances of `.template-card` global-class, so this style will leak into other components/pages that have this global-class. I would move this into the `.cedar-container` local-class's style to avoid any unexpected styling outside of this component